content/blog/oddnews/oddnews-21.md

```diff
@@ -0,0 +1,198 @@
+---
+title: CSS Cascade Layers
+issue: 21
+url: http://eepurl.com/i_NTYw
+date: 2025-03-13
+tags:
+  - CSS
+  - Anchor Positioning
+  - Typography
+length: 3 mins
+image:
+  src: winging-it/winging-it-1.jpg
+summary: |
+  Here’s what we’re up to:
+  - Revisiting Fluid Type on Winging It with Richard Rutter | March 20, 2025
+  - New features for OddContrast
+  - CSS Layout Workshop coming in April 2025
+  - Custom CSS Functions in the Browser article by Miriam Suzanne
+---
+
+{% import "utility.macros.njk" as utility %}
+
+{{ utility.main_action(
+  'Subscribe to OddNews »',
+  subscribe_url
+) }}
+
+## Upcoming Winging It LIVE
+
+[Revisiting Fluid Type](https://www.youtube.com/watch?v=py41Ys-iRvk) with
+[Stacy Kvernmo](/authors/stacy), [Miriam Suzanne](/authors/miriam), and
+guest Richard Rutter on March 20, 2025.
+
+[Save the date](https://www.youtube.com/watch?v=py41Ys-iRvk)
+
+## OSS Updates
+
+### New Features for OddContrast
+
+If you haven’t checked it out, [OddContrast](/oddcontrast/) is OddBird’s color
+format converter, with newer color formats like Oklch, Oklab, and the Display P3
+color space. It’s also a color contrast checker to help designers meet WCAG 2
+accessibility standards.
+
+- Swap background and foreground colors. Either drag-and-drop or use the button
+  located between the two colors.
+- A new Show Gamut dropdown menu allows displaying color gamut ranges on the
+  color sliders. And we’ve made it more obvious when a color is outside the
+  specified gamut.
+- Contrast ratios now incorporate foreground color alpha values. Ratios
+  displayed are an estimation of contrast when the foreground color is not
+  opaque.
+
+### from [James Stuckey Weber](/author/james/)
+
+I reviewed feature changes for the Web Features project and contributed to the
+Web DX Community Group. I also contributed a bug fix to the Firefox Dev Tools to
+show the offsetParent of fixed position elements if it’s not the viewport. It
+should be included in the Firefox 137 release.
+
+{% callout 'note' %}
```

Comment:
This doesn't feel right to me as a callout under the 'James' heading. I think it should get it's own section in a blog post like this one.